### PR TITLE
feat: add odds calculator

### PR DESCRIPTION
**Pull request by anddigital-olikeighley**

### CODE REVIEW

File: src/utils/oddsCalculator.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * @description Returns decimal odds given numerator and denominator of fractional odds.
+ * @arg {number} numerator
+ * @arg {number} denominator
+ * @arg {number} decimalPlaces
+ * @example
+ * // return 1.33
+ * convertFractionalToDecimal(1, 3)
+ */
+export const convertFractionalToDecimal = (numerator, denominator, decimalPlaces = 2) => {
+  const result = (numerator + denominator) / denominator;
```
Comment (DavidChristopherJohnson) [inline src/utils/oddsCalculator.js:11]: The calculation here is wrong it should be:

const result = numerator / denominator. 

Adding the denominator to the numerator effectively gives you the answer + 1

Comment (anddigital-olikeighley) [reply src/utils/oddsCalculator.js:11]: The calculation is correct. I'm on my phone rn but Google it and you will see 1/3 on a betting website would be 1.33 decimal.

Comment (anddigital-olikeighley) [reply src/utils/oddsCalculator.js:11]: Otherwise you would lose money on odds of 1/3. 0.33 * your stake makes no sense.

**Review by mangohatface — APPROVED** [review]
Looks good to me :shipit:

(though, a little unsure why you needed to force push one of your commits 😅  )

**Pull request merged**